lib/pc_util.c

```diff
@@ -263,3 +288,80 @@ void pc_bounds_merge(PCBOUNDS *b1, const PCBOUNDS *b2)
 	if ( b2->ymax > b1->ymax ) b1->ymax = b2->ymax;
 }
 
+uint8_t *
+pc_bounds_xyzm_to_bounding_diagonal_wkb(
+	const PCSTATS *stats, const PCBOUNDS *bounds, const PCSCHEMA *schema,
+	size_t *wkbsize)
```

Comment:
it is a little cumbersome to pass both the stats and the bounds.
Is it much more efficient to use the bounds rather than the stats only ?
schema is already accessible in the points of stats.